### ISSUE TITLE
environmentVariables support post kotlin rewrite?

### ISSUE THREAD

**Issue by huggsboson** [unbroken-dome/gradle-testsets-plugin#81]
Trying to use environmentVariables with the 2.1.1 version of the plugin seems to be breaking.

e.g.
```
Could not set unknown property 'environmentVariables' for object of type org.unbrokendome.gradle.plugins.testsets.dsl.DefaultTestSet.
```

I notice there's an integration test inside the plugin written in groovy.  I tried running './gradlew integrationTest' on a checkout of this repo and it doesn't seem to be run.  Are we planning on reimplementing this?

**Comment by tkrullmann** [comment]
Thanks for the report, I must have overlooked `environmentVariables` and `systemProperties` in the rewrite. I will bring them back soon :-)

**Comment by tkrullmann** [comment]
The feature is now available as part of the 2.2.0 release.
Note, I used the name `environment` instead of `environmentVariables`, to stay consistent with what the properties are called in the `Test` / `JavaForkOptions` classes from Gradle.

**Comment by huggsboson** [comment]
@tkrullmann thank you so much for the very fast turn around.  You're a beast!
